### PR TITLE
Omar model reserve vehicle

### PR DESCRIPTION
**Pull request by OmarOporto3**
All CRUD for model, vehicle, reserve

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +18,8 @@
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "nodemon": "^3.0.1"
```
Comment (JoseLozadaAD) [inline package.json:22]: Why you add nodemon ? this have to go in another PR

File: src/controllers/models.controller.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import {
+  getAllModelsRepository,
+  getSingleModelRepository,
+  createModelRepository,
+  updateModelRepository,
+  deleteModelRepository
+} from '../repositories/model.repository.js'
+
+export const getAllModels = async (req, res) => {
```
Comment (JoseLozadaAD) [inline src/controllers/models.controller.js:9]: Change req and res variables because are not descriptives, do this in every req, res variable

File: package.json (hunk 1 above)
Comment (jacovzap-assuresoft) [reply package.json:22]: It's ok but it should be in dev dependencies

Comment (OmarOporto3) [reply package.json:22]: Yes...Sorry i didn't notice that

File: src/controllers/models.controller.js
Comment (Juan-Pablo-Chavez-AS) [inline src/controllers/models.controller.js:1]: Should be model.controller.js

File: src/controllers/models.controller.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  try {
+    const addresses = await getAllModelsRepository()
```
Comment (Juan-Pablo-Chavez-AS) [inline src/controllers/models.controller.js]: Should be Models, not addresses?

File: src/controllers/reserve.controller.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  getAllReservesRepository,
+  getSingleReserveRepository,
+  createReserveRepository,
+  updateReserveRepository,
+  deleteReserveRepository
+} from '../repositories/reserve.repository.js'
+
+export const getAllReserves = async (req, res) => {
+  try {
+    const addresses = await getAllReservesRepository()
```
Comment (Juan-Pablo-Chavez-AS) [inline src/controllers/reserve.controller.js]: Should be Models, not addresses
